pkg/config/config.go

```diff
@@ -16,6 +16,12 @@ const (
 	configFileName = "odo"
 )
 
+// OdoSettings holds all odo specific configurations
+type OdoSettings struct {
+	// holds the status for update checker
```

Comment:
see my comment for `odo set` command description

Reply:
@kadel  `odo set` ??

Reply:
okey got it https://github.com/redhat-developer/odo/pull/589#discussion_r209836427